nightfall-client/src/routes/commitment.mjs

```diff
@@ -82,6 +97,24 @@ router.get('/commitments', async (req, res, next) => {
   }
 });
 
+/**
+ * @description the endponit that will send a reponse with all the
+ * existent commitments if the query param compressedPkd comes undefined
+ * or all commitments by compressedPkd if the query param comes with a string.
+ * @author luizoamorim
+ */
+router.get('/all', async (req, res, next) => {
+  logger.debug('commitment/all endpoint received GET');
+  const { compressedPkd } = req.query;
+  try {
+    const commitments = await getCommitmentsByCompressedPkd(compressedPkd);
+    res.json({ commitments });
+  } catch (err) {
+    logger.error(err);
+    next(err);
+  }
+});
```

Comment:
Since this is the beginning of a new era (I do not dislike the idea, let's see what the rest have to say), why don't we create a file `commitments` (note the `s` at the end). Then this route could simply be `/commitments`.

**What do you think?**